src/install-script.nsi

```diff
@@ -0,0 +1,417 @@
+; Script generated by the HM NIS Edit Script Wizard.
+Unicode True
+
+; HM NIS Edit Wizard helper defines
+!define PRODUCT_NAME "Structuraid"
+!define PRODUCT_VERSION "0.1.0"
+!define PRODUCT_PUBLISHER "PradaIng"
+!define PRODUCT_WEB_SITE "https://github.com/pradaing"
+!define PRODUCT_DIR_REGKEY "Software\Microsoft\Windows\CurrentVersion\App Paths\elevate.exe"
+!define PRODUCT_UNINST_KEY "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT_NAME}"
+!define PRODUCT_UNINST_ROOT_KEY "HKLM"
+
+; MUI 1.67 compatible ------
+!include "MUI.nsh"
+
+; MUI Settings
+!define MUI_ABORTWARNING
+!define MUI_ICON "assets\structuraid-icon.ico"
+!define MUI_UNICON "${NSISDIR}\Contrib\Graphics\Icons\modern-uninstall.ico"
+
+; Welcome page
+!insertmacro MUI_PAGE_WELCOME
+; License page
+!insertmacro MUI_PAGE_LICENSE "..\LICENSE"
+; Directory page
+!insertmacro MUI_PAGE_DIRECTORY
+; Instfiles page
+!insertmacro MUI_PAGE_INSTFILES
+; Finish page
+!define MUI_FINISHPAGE_RUN "$INSTDIR\UI\Structuraid Desktop.exe"
+!insertmacro MUI_PAGE_FINISH
+
+; Uninstaller pages
+!insertmacro MUI_UNPAGE_INSTFILES
+
+; Language files
+!insertmacro MUI_LANGUAGE "English"
+
+; MUI end ------
+
+Name "${PRODUCT_NAME} ${PRODUCT_VERSION}"
+OutFile "Structuraid.exe"
+InstallDir "$PROGRAMFILES64\Structuraid"
+InstallDirRegKey HKLM "${PRODUCT_DIR_REGKEY}" ""
+ShowInstDetails show
+ShowUnInstDetails show
+
+Section "MainSection" SEC01
+  SetOutPath "$INSTDIR"
+  File "..\dist\.keep"
```

Comment:
This is copying every single file in the `dist/` folder, 1 by 1. Not sure if there is a way to copy the whole folder in a single command 😬

Reply:
Looks like it's possible. Will test this out later
https://stackoverflow.com/questions/11014013/copy-a-directory-using-nsis

Reply:
🥹